libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/saveDashboardHandler.test.ts

```diff
@@ -135,32 +135,4 @@ describe("getDashboardWithSharing", () => {
             expect(isUnderStrictControl).toBe(expectedResult.isUnderStrictControl);
         },
     );
-
-    it.each([
```

Comment:
Should we skip and migrate later?

Reply:
Locked is not supported in this way ... on tiger is done via parent workspace thats why deleted